crates/cairo-coverage/src/input/test_function_filter.rs

```diff
@@ -0,0 +1,27 @@
+use crate::types::FunctionName;
+use cairo_lang_sierra::ids::FunctionId;
+
+pub struct TestFunctionFilter {
+    include_test_functions: bool,
+    test_functions: Vec<String>,
+}
+
+impl TestFunctionFilter {
+    pub fn new(
+        functions_marked_with_test_attribute: Option<&Vec<FunctionId>>,
```

Comment:
[nit]: i'd rather have 2 constructors, one with, and one without it

Reply:
Changed it to take &[FunctionId]